internal/configs/version2/nginx-plus.virtualserver.tmpl

```diff
@@ -1,6 +1,6 @@
 {{ range $u := .Upstreams }}
 upstream {{ $u.Name }} {
-    zone {{ $u.Name }} 256k;
+    {{if ne $u.UpstreamZoneSize "0"}}zone {{$u.Name}} {{$u.UpstreamZoneSize}}; {{ else }} zone {{$u.Name}} 256k;{{end}}
```

Comment:
same problem the the brackets usage again -- https://github.com/nginxinc/kubernetes-ingress/pull/659#discussion_r314012813